unicorn_contracts/src/contracts_service/updateContractFunction.ts

```diff
@@ -34,103 +26,101 @@ class UpdateContractFunction implements LambdaInterface {
      * @returns {Object} object - API Gateway Lambda Proxy Output Format
      *
      */
-     @tracer.captureLambdaHandler()
-     @metrics.logMetrics()
-     @logger.injectLambdaContext({ logEvent: true })
+    @tracer.captureLambdaHandler()
+    @metrics.logMetrics({ captureColdStartMetric: true })
+    @logger.injectLambdaContext({ logEvent: true })
```

Comment:
I left this in, however we recommend to use it only for development.

Reply:
Good to help people understand the feature. I will make sure we comment this across the board. Maybe an idea to make this conditional based on the log level? 🤷🏻